packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -307,6 +307,20 @@ const launchRole = new iam.Role(this, 'LaunchRole', {
 portfolio.setLaunchRole(product, launchRole);
 ```
 
+You can also set the launch role to use a role of a given name that exists in the local account. This can be useful when sharing a portfolio with multiple accounts, as
+the launch role in the launch role constraint will be account-agnostic resulting in fewer resources needing to be created per shared account.
+A role with the given role name must exist in the account that created the launch constraint and the account of the user who launches a product with this launch constraint.
+
+```ts fixture=portfolio-product
+import * as iam from '@aws-cdk/aws-iam';
+
+const launchRole = new iam.Role(this, 'LaunchRole', {
+  assumedBy: new iam.ServicePrincipal('servicecatalog.amazonaws.com'),
+});
+
+portfolio.setLocalLaunchRoleName(product, 'LaunchRole');
```

Comment:
See [here](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/constraints-launch.html#constraints-launch-constraint) for more about this part of the feature. I know we generally don't like to reference primitive types but i think it makes sense in this case

Reply:
I have to say, I don't follow the example. `'LaunchRole'` is the logical ID, not the physical name of the Role... I think you wanted something like this?

```ts
const roleName = 'MyRole';
const launchRole = new iam.Role(this, 'LaunchRole', {
  assumedBy: new iam.ServicePrincipal('servicecatalog.amazonaws.com'),
  roleName,
});
portfolio.setLocalLaunchRoleName(product, roleName);
```

Reply:
yep that is my bad